cmd/globalAlignment/globalAlignment.go

```diff
@@ -94,37 +68,62 @@ func main() {
 		outFile.Close() //commented out defer outFile.Close()
 	}
 
-	//graph
-	genomeGraph := cigarToGraph(faOne, faTwo, aln)
-	simpleGraph.PrintGraph(genomeGraph)
-
-}
-
-//faOne target, faTwo query
-//cigar to graph takes two fasta files as an input, performs a global alignment, and produces genome graph
-func cigarToGraph(target *fasta.Fasta, query *fasta.Fasta, aln []align.Cigar) *simpleGraph.SimpleGraph {
-	answer := simpleGraph.NewGraph()
+	//cigar to graph
+	genomeGraph := simpleGraph.NewGraph()
 	//use targetEnd and queryEnd to track position number for each fasta sequence as we move along.
 	var targetEnd, queryEnd int = 0, 0
 	var curr *simpleGraph.Node
 	//TODO: Make sure it can handle non-match at node 0. Fill out annotations for each node (knowing which allele/haplotype).
 	//TODO: Handle multi-entry fasta?
-
 	//Creating the first node. This is done independent of all other number because this node has no 'previous' nodes.  All following code leverages the cigar output (second number printed in the Op position of the struct {}) to determine if the alignment returned a 0:match, 1:insertion, or 2:deletion. All indels are relative to the target sequence.
-	curr, targetEnd, queryEnd = simpleGraph.FaSeqToNode(target, query, targetEnd, queryEnd, aln[0], 0)
-	simpleGraph.AddNode(answer, curr)
+	curr, targetEnd, queryEnd = simpleGraph.FaSeqToNode(faOne, faTwo, targetEnd, queryEnd, aln[0], 0)
+	simpleGraph.AddNode(genomeGraph, curr)
 	//Drawing the remaining nodes and all edges. Method for adding edges is based on previous nodes.
 	for i := 1; i < len(aln); i++ {
-		curr, targetEnd, queryEnd = simpleGraph.FaSeqToNode(target, query, targetEnd, queryEnd, aln[i], i)
-		simpleGraph.AddNode(answer, curr)
+		curr, targetEnd, queryEnd = simpleGraph.FaSeqToNode(faOne, faTwo, targetEnd, queryEnd, aln[i], i)
+		simpleGraph.AddNode(genomeGraph, curr)
 		if aln[i].Op == 0 {
-			simpleGraph.AddEdge(answer.Nodes[i-1], curr, 1)
-			simpleGraph.AddEdge(answer.Nodes[i-2], curr, 0.5)
+			simpleGraph.AddEdge(genomeGraph.Nodes[i-1], curr, 1)
+			simpleGraph.AddEdge(genomeGraph.Nodes[i-2], curr, 0.5)
 		} else if aln[i].Op == 1 || aln[i].Op == 2 {
-			simpleGraph.AddEdge(answer.Nodes[i-1], curr, 0.5)
+			simpleGraph.AddEdge(genomeGraph.Nodes[i-1], curr, 0.5)
 		} else {
 			log.Fatalf("Error: cigar.Op = %d is unrecognized...\n", aln[i].Op)
 		}
 	}
-	return answer
+	simpleGraph.PrintGraph(genomeGraph)
+}
+
+//raven edited this block to specify only 1 sequnce is expected in each fasta file and add Usage nad options
+func usage() {
+	fmt.Print(
+		"./globalAlignment - chelsea's global alignment\n" +
+			" Align 2 .fasta files, each with only 1 sequence\n" +
+			"Usage:\n" +
+			"	faTarget faQuery\n" +
```

Comment:
faTarget and faQuery are the input variables. Would it make more sense to label line 102 as input variable instead of usage?

Reply:
Yes you are right. They are input variables. It seems from the other examples in the repository that "usage" is followed by what users type on the command line to run the program, so that's why I wrote the input variables on line 103

Reply:
that's true, but it should also have the executable's name so it would be 'globalAlignment faTarget faQuery' (could also expand to be 'globalAlignment target.fasta query.fasta')

Reply:
Got it! Just fixed. Thanks!